boards/adafruit-pybadge/doc.txt

```diff
@@ -0,0 +1,33 @@
+/**
+@defgroup    boards_adafruit-pybadge Adafruit PyBadge
+@ingroup     boards
+@brief       Support for the Adafruit PyBadge board.
+
+### General information
+
+The [Adafruit PyBadge](https://www.adafruit.com/product/4200) and
+[Adafruit PyBadge LC](https://www.adafruit.com/product/3939) are
```

Comment:
I'm missing indication here whether these are supported by the same board, and if so, whether it's autodetected or needs to be configured. (Original post says "should work", but that could mean "just work" or "be extensible to").

Reply:
I extended the documentation to try to make this clearer.